Translations update from Kodi Weblate

weblate opened the pull request:
Translations update from [Kodi Weblate](https://kodi.weblate.cloud) for [Kodi add-ons: video/plugin.video.youtube](https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/).



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-add-ons-video/-/plugin-video-youtube/horizontal-auto.svg)

gade01 reviewed (approved):
Translated using translation memory from other components.

weblate committed:
Translated using Weblate (Czech (cs_cz))

Currently translated at 100.0% (300 of 300 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 100.0% (300 of 300 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (300 of 300 strings)

Translated using Weblate (Vietnamese (vi_vn))

Currently translated at 5.6% (17 of 300 strings)

Translated using Weblate (Swedish (sv_se))

Currently translated at 3.3% (10 of 300 strings)

Translated using Weblate (Belarusian (be_by))

Currently translated at 0.3% (1 of 300 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 41.3% (124 of 300 strings)

Translated using Weblate (Portuguese (Portugal) (pt_pt))

Currently translated at 94.0% (282 of 300 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 96.0% (288 of 300 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 95.6% (287 of 300 strings)

Translated using Weblate (Norwegian Bokmål (nb_no))

Currently translated at 43.3% (130 of 300 strings)

Translated using Weblate (Hebrew (Israel) (he_il))

Currently translated at 58.0% (174 of 300 strings)

Translated using Weblate (English (United States) (en_us))

Currently translated at 3.3% (10 of 300 strings)

Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kryštof Černý <cleverline1mc@gmail.com>
Co-authored-by: Oskari Lavinto <olavinto@protonmail.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/be_by/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/cs_cz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/en_us/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/he_il/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/nb_no/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/nl_nl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/pt_br/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/pt_pt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/ro_ro/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/sv_se/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/vi_vn/
Translation: Kodi add-ons: video/plugin.video.youtube
